recipes/fruit/all/conandata.yml

```diff
@@ -8,21 +8,44 @@ sources:
   "3.6.0":
     url: "https://github.com/google/fruit/archive/v3.6.0.tar.gz"
     sha256: "b35b9380f3affe0b3326f387505fa80f3584b0d0a270362df1f4ca9c39094eb5"
+  "3.7.0":
```

Comment:
Ideally new version are on top (thats the order the build service works which leads to faster results)

Reply:
I would propose to close this PR with this change the way it is and the  when I add fruit 3.7.1, I will adapt accordingly. Agreed?

Reply:
Absolutely, not a blocker just an FYI ❤️ just for whenever the next PR is, it's not worth one on it's own